docs/components/InputText/Web.stories.tsx

```diff
@@ -27,6 +28,31 @@ Multiline.args = {
   rows: { min: 1, max: 5 },
 };
 
+export const Toolbar = BasicTemplate.bind({});
+Toolbar.args = {
+  placeholder: "Hakunamatata",
+  multiline: true,
+  rows: { min: 1, max: 5 },
+  toolbar: (
+    <div
+      style={{
+        display: "flex",
+        width: "100%",
+        justifyContent: "space-between",
+      }}
+    >
+      <Button label="Rewrite" size="small" icon="sparkles" fullWidth={false} />
+      <Button
+        ariaLabel="Undo"
+        size="small"
+        icon="redo"
+        type="tertiary"
+        fullWidth={false}
+      />
+    </div>
```

Comment:
i think we can just put them side by side instead of very far away. UX wise, that's better

Reply:
fair, but I wanted to make sure to have an example that had a button in the bottom right because that's a design we've seen, and I had worries about how close it would end up being to the resize handler

so having an example with that layout lets us try it out

Reply:
oh, that makes sense!